### PR TITLE
[Community Tutorials] Tweak "Custom base map" tutorial for publication

### PR DESCRIPTION
**Pull request by gino-m**

### CODE REVIEW

**Comment by gino-m** [comment]
@jdbcode could you PTAL?

@TC25 are you ok with these edits?

File: tutorials/customizing-base-map-styles/index.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 tags: Earth Engine, Base Map, UI, Widgets
 date_published: 2019-10-16
 ---
 
 [Open In Code Editor](https://code.earthengine.google.com/283c86dd3189788452b5fb64174a1b43)
 
-The user is going to learn how to change the options of the map object to create
-new base maps
+In this tutorial, you'll learn how to change the options of the `Map` object to
+define your our styles for the unlying base map.
```
Comment (jdbcode) [inline tutorials/customizing-base-map-styles/index.md]: "unlying" > "underlying" (?)

File: tutorials/customizing-base-map-styles/index.md [hunk 2]
```diff
@@ -48,17 +48,17 @@ The main styler options include:
...
 https://developers.google.com/maps/documentation/javascript/style-reference
 
 The full list of features (also in the Google Maps API documentation linked
-above) includes geometries and labels (including their texts and icons). All
-styler options work with each of these features.
+above) includes geometries, labels, icons, and more. All styler options work
+with each of these features.
 
-Let's consider a couple of examples to edit other elements and features
+Let's consider a few examples:
```
Comment (jdbcode) [inline tutorials/customizing-base-map-styles/index.md]: Suggestion: remove "Let's"

File: tutorials/customizing-base-map-styles/index.md [hunk 3]
```diff
@@ -211,11 +211,13 @@ Map.setOptions('roadNetwork', {
...
-There is a way to create almost any base map you want without the hassle of
-tweaking any options. Enter [Snazzy Maps](https://snazzymaps.com), a community
-project for creating and sharing great styles for Google Maps. Copy any of the
-javascript properties from that website and it will show up on Earth Engine.
-Consider the two examples below:
+There is also an easy way to create custom base map styles without
+tweaking any options: enter [Snazzy Maps](https://snazzymaps.com), a community
+project for creating and sharing great styles for Google Maps. Their website
+provides JavaScript snippets that can be copied from their website and pasted into
+Earth Engine to quickly create beautiful base map styles.
```
Comment (jdbcode) [inline tutorials/customizing-base-map-styles/index.md]: Suggestion: "beautiful" > "alternative"

File: tutorials/customizing-base-map-styles/index.md (hunk 1 above)
Comment (gino-m) [reply tutorials/customizing-base-map-styles/index.md]: Funny, thought I fixed this.

**Comment by gino-m** [comment]
Updated! PTAL.

**Comment by gino-m** [comment]
> Note that I edited spelling on line 222.

Thanks for the fix.

**Comment by gino-m** [comment]
@TC25 these are just a few tweaks to the tutorial to more closely match https://developers.google.com/style. Let me know if it looks ok to you and I'll proceed to publish.

**Comment by TC25** [comment]
@gino-m I suggested an edit on line 12. Otherwise, looks good to me. 

Also, found a non-programmatic way to create map styles (https://mapstyle.withgoogle.com/) Should I add a link to this there?

**Comment by gino-m** [comment]
Sure! Would you like to send a small PR to add these? I'll look at it right
away without additional technical review.

**Pull request merged**